main.go

```diff
@@ -28,6 +28,7 @@ func main() {
 	extensionsFlag := flag.String("extensions", "html,php", "comma separated list of extensions to append")
 	queueSize := flag.Int("queue-size", 5000, "number of urls that can sit in the queue at one time")
 	pollerBatchSize := flag.Int("poller-batch-size", 5000, "number of urls the poller can pull from the database in one go")
+	timeOut := flag.Int("time-out", 10, "HTTP timeout in seconds")
```

Comment:
Can we change the flag to `timeout` since it's just one word throughout the Go codebase by the looks of it. I've also gone with all lowercase for the descriptions so can we make it

"http timeout in seconds, specify zero for no timeout"

Also sorry to be nit-picky, but can we also drop the camel casing for the variable if we're treating it as one word.

Lastly, any idea what happens when we specify timeout values less than zero? Might be worth adding a check to validate it.